pkg/utils/sets/sets.go

```diff
@@ -45,11 +47,28 @@ func NewComplementSet(values ...string) Set {
 	}
 }
 
+// Hash provides a hash function so we can generate a good hash for Set which has no public fields.
+func (s Set) Hash() (uint64, error) {
```

Comment:
Is this still used?

Reply:
No, but I left it in thinking we might hash something with a Set() later.  If we did, the hash library would just ignore the Set type and not issue an errors.